### PR TITLE
Removing PSAzureFirewallPolicyThreatIntelWhitelist validations

### PR DESCRIPTION
**Pull request by amosarbiv-work**
Threat intel model is not able to be deserialized because of the validation which lack validations for ip ranges and CIDR.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by isra-fel** [comment]
Hey @amosarbiv-work your ChangeLog.md has conflicts with main branch. Please resolve them so CI can be unblocked. Thanks

File: src/Network/Network/ChangeLog.md
Comment (gersner) [inline src/Network/Network/ChangeLog.md:22]: Unrelated(?)

Comment (amosarbiv-work) [reply src/Network/Network/ChangeLog.md:22]: I merged master, I think that's an issue with the merging :/

Comment (gersner) [reply src/Network/Network/ChangeLog.md:22]: Dont see it in master, remove it

**Review by gersner — APPROVED** [review]

File: src/Network/Network/ChangeLog.md
Comment (amosarbiv-work) [reply src/Network/Network/ChangeLog.md]: done

**Review by gersner — APPROVED** [review]

**Comment by amosarbiv-work** [comment]
@MikhailTryakhov , please review :)

**Review by isra-fel — APPROVED** [review]
LGTM. Let me know if this is still pending on @MikhailTryakhov 's review.

**Comment by amosarbiv-work** [comment]
@isra-fel , I dont understand your comment? why is the Do-Not-Merge label was added?! Thanks. I'm still waiting for a review

**Comment by isra-fel** [comment]
Hey @amosarbiv-work , I saw you explictly requested review from Mikhail so I just want to make sure if you want to hold on the PR until he approves, so I added the lable.

IMO this is good to go.

**Comment by amosarbiv-work** [comment]
@isra-fel , so can you please remove the labels so I can merge? I've added my initial comment because I thought Mikhail was the assigned engineer for the review. If you have enough permissions and think the PR is GTG I would appreciate merging it. Thank you very much!

**Pull request merged**
